.env.example

```diff
@@ -23,7 +23,9 @@ EMAIL_TEST_ADDRESS=""
 EMAIL_SERVICE_EMAIL=""
 
 # Renewal emails config
+SECOND_RENEWAL_EMAIL_BEFORE_DAYS=14
 FIRST_RENEWAL_EMAIL_REMINDER_DAILY_RUN_TIME="1:05"
+SECOND_RENEWAL_EMAIL_REMINDER_DAILY_RUN_TIME="1:05"
```

Comment:
See later comment where I have screwed up the time (🤦‍♂) but our default for this job is now "04:05" in the schedule, so would be great to update the `.env.example` to match.

Reply:
```suggestion
SECOND_RENEWAL_EMAIL_REMINDER_DAILY_RUN_TIME="04:05"
```